src/app/public/plugin-resources/code-examples/list-view-grid/templated-column/list-view-grid-demo.component.scss

```diff
@@ -0,0 +1,6 @@
+@import "~@skyux/theme/scss/mixins";
+@import "~@skyux/theme/scss/_compat/mixins";
```

Comment:
We probably don't want to expose the `_compat` styles to our consumers.

Reply:
Removed.